golem-common/src/model/component.rs

```diff
@@ -76,12 +77,6 @@ impl FromStr for DefaultComponentOwner {
     }
 }
 
-impl HasAccountId for DefaultComponentOwner {
```

Comment:
IMO this guy is guy confusing when it shows up in constraints, instead I inlined it into the higher-level traits